public/app/ask/ask.html

```diff
@@ -0,0 +1,7 @@
+<div ng-controller="AskController">
+  <div class="userDisplay story" ng-repeat="story in stories | limitTo: index">
+    <div ng-include="'app/partials/story.html'" class="story-holder"></div>
+  </div>
+
+  <div class="more-button" ng-show="index < stories.length && stories.length > 0" ng-click="index = index + perPage;">more</div>
```

Comment:
Yeah, but I didn't want to clobber it and cause a conflict. Gonna do a quick branch after this to fix it. Alternatively, I can rebase it now and just make the changes.